doc/system_health_monitoring/system-health-HLD.md

```diff
@@ -55,6 +55,9 @@ By default any service is not in expected status will be considered as fault con
 
 -  Any fan is missing/broken
 -  Fan speed is lower than minimal value
+-  Fan direction is not expected
+    - All fan direction must be the same
```

Comment:
same direction for ALL fans across all planforms?

Reply:
Same direction for the same platform